release_notes.html

```diff
@@ -0,0 +1,18 @@
+<b>BerryIO Release Notes - Published by Splunk February 07, 2018</b>
+<br><br>
+<b>Version 1.0.8 - Released February 07, 2018</b>
+<ul>
+<li>App action views and Logo updates</li>
```

Comment:
We already released this release note, but `Logo` can be `logo

Reply:
done